store/actions.js

```diff
@@ -0,0 +1,49 @@
+import axios from "axios"
+export default {
+	// May be a way to consolidate fetchPage and fetchPosts into one action
+	fetchPages: async (context) => {
```

Comment:
Consolidate `fetchPages` and `fetchPosts` into one function; add page- or post-specific logic in conditionals based on post type. We can look at this together if that would be helpful.

Reply:
Yes please, looking at it together will be best.